PKHeX/Saves/SAV1.cs

```diff
@@ -0,0 +1,522 @@
+        public override ushort TID
+        {
+            get { return Util.SwapEndianness(BitConverter.ToUInt16(Data, Japanese ? 0x25FB : 0x2605)); }
+            set { BitConverter.GetBytes(Util.SwapEndianness(value)).CopyTo(Data, Japanese ? 0x25FB : 0x2605); }
+        }
+        public override ushort SID
+        {
+            get { return 0; }
+            set { }
+        }
+        public override int PlayedHours
+        {
+            get { return BitConverter.ToUInt16(Data, Japanese ? 0x2CA0 : 0x2CED); }
+            set { BitConverter.GetBytes((ushort)value).CopyTo(Data, Japanese ? 0x2CA0 : 0x2CED); }
+        }
+        public override int PlayedMinutes
+        {
+            get { return Data[Japanese ? 0x2CA2 : 0x2CEF]; }
+            set { Data[Japanese ? 0x2CA2 : 0x2CEF] = (byte)value; }
+        }
+        public override int PlayedSeconds
+        {
+            get { return Data[Japanese ? 0x2CA3 : 0x2CF0]; }
+            set { Data[Japanese ? 0x2CA3 : 0x2CF0] = (byte)value; }
+        }
+
+        public int Badges
+        {
+            get { return Data[Japanese ? 0x25F8 : 0x2602]; }
+            set { if (value < 0) return; Data[Japanese ? 0x25F8 : 0x2602] = (byte)value; }
+        }
+
+        private byte Options
+        {
+            get { return Data[Japanese ? 0x25F7 : 0x2601]; }
+            set { Data[Japanese ? 0x25F7 : 0x2601] = value; }
+        }
+
+        public bool BattleEffects
+        {
+            get { return (Options & 0x80) == 0; }
+            set { Options = (byte)((Options & 0x7F) | (value ? 0 : 0x80)); }
+        }
+
+        public bool BattleStyleSwitch
+        {
+            get { return (Options & 0x40) == 0; }
+            set { Options = (byte)((Options & 0xBF) | (value ? 0 : 0x40)); }
+        }
+
+        public int Sound
+        {
+            get { return (Options & 0x30) >> 4; }
+            set
+            {
+                var new_sound = value;
+                if (new_sound > 3)
+                    new_sound = 3;
+                if (new_sound < 0)
+                    new_sound = 0;
+                Options = (byte)((Options & 0xCF) | (new_sound << 4));
+            }
+        }
+
+        public int TextSpeed
+        {
+            get { return Options & 0x7; }
+            set
+            {
+                var new_speed = value;
+                if (new_speed > 7)
+                    new_speed = 7;
+                if (new_speed < 0)
+                    new_speed = 0;
+                Options = (byte)((Options & 0xF8) | new_speed);
+            }
+        }
+
+        public override uint Money
+        {
+            get { return uint.Parse((Util.SwapEndianness(BitConverter.ToUInt32(Data, Japanese ? 0x25EE : 0x25F3)) >> 8).ToString("X6")); }
+            set
+            {
+                BitConverter.GetBytes(Util.SwapEndianness(uint.Parse(value.ToString("000000"), NumberStyles.HexNumber))).Skip(1).ToArray().CopyTo(Data, Japanese ? 0x25EE : 0x25F3);
```

Comment:
Yeah, seems simple enough. 

A comment in the src would be great; the game stores this as a Binary Coded decimal' would prevent confusion for anyone using this code centuries from now :)